defaults/main.yaml

```diff
@@ -12,6 +12,15 @@ clamav_epel_yumrepo_gpgkey: "{{ yumrepo_epel_gpgkey | default('https://dl.fedora
 # Additional EPEL YUM repo params
 clamav_epel_yumrepo_params: "{{ yumrepo_epel_params | default({}) }}"
 
+# In the event of network congestion
+clamav_network_max_retries: 0
+
+# ClamAV operating as client-only
+clamav_client_only: false
```

Comment:
I would create two switches `clamav_client` and `clamav_server` because you might want to run client on the server as well and in that case the variable name is misleading.

Reply:
AFAIK, server implies client. Is there a configuration case in `/etc/clamd.conf` in which the ClamAV server runs locally and clamdscan/clamscan cannot run?